client/selectors/index.js

```diff
@@ -308,3 +308,17 @@ export const isAdvancedDateSearch = createSelector(
         !!get(currentSearch, 'advancedSearch.dates.range', false)
     )
 )
+
+/** Returns the list of currently locked planning items */
+export const getLockedPlannings = (state) => filter(state.planning.plannings, matches(
```

Comment:
Use `createSelector([getStoredPlannings, getCurrentUserId], (plannings, userId) => ....` when creating this selector for better optimization.